homeassistant/components/google_generative_ai_conversation/entity.py

```diff
@@ -301,14 +302,13 @@ async def _transform_stream(
 class GoogleGenerativeAILLMBaseEntity(Entity):
     """Google Generative AI base entity."""
 
-    _attr_has_entity_name = True
-    _attr_name = None
-
-    def __init__(self, entry: ConfigEntry) -> None:
+    def __init__(self, entry: ConfigEntry, subentry: ConfigSubentry) -> None:
         """Initialize the agent."""
         self.entry = entry
+        self.subentry = subentry
+        self._attr_name = subentry.title or DEFAULT_CONVERSATION_NAME
```

Comment:
It doesn't seem title is ever empty. It's required in the subentry flow and it's set to DEFAULT_CONVERSATION_NAME in the main entry flow as well as the migrate flow.

Reply:
People can rename config subentries

Reply:
Do we allow empty titles when we rename?